sphinx/conf.py

```diff
@@ -20,7 +20,7 @@
 import sys
 
 sys.path.insert(0, os.path.abspath('..'))
-import aodncore.version
+import aodncore._version
```

Comment:
Similarly here...
```suggestion
from aodncore import __version__
```
... and update below accordingly.